public/JS Files/main.js

```diff
@@ -0,0 +1,55 @@
+/*------Functions------*/
+function capitalize(word) {
+    let steralize = word.toString().trim().toLowerCase()
+    let firstLetter = steralize[0].toUpperCase()
+    let restOfWord = steralize.slice(1)
+    return firstLetter + restOfWord
+}
+
+function titleize(sentence) {
+    let wordArray = sentence.toLowerCase().split(' ')
+    let newArray = wordArray.map(capitalize)
+    return newArray.join(' ')
+}
+
+let restList = document.getElementById('restList')
+
+/*----------Restaruant Map Location and Details--------*/
+async function restaurantDetails() {
```

Comment:
Good! Don't forget to add some comments throughout here so we know what is happening in each of these fetches. Also, now that this is working, can you think of a way to be able to pull some of this functionality into separate functions?